docs/examples/data_connectors/MongoDemo.ipynb

```diff
@@ -84,6 +85,55 @@
             "source": [
                 "display(Markdown(f\"<b>{response}</b>\"))"
             ]
+        },
+        {
+            "attachments": {},
+            "cell_type": "markdown",
+            "id": "1671858a",
+            "metadata": {},
+            "source": [
+                "## MongoDB Atlas"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 3,
+            "id": "50f4e616",
+            "metadata": {},
+            "outputs": [
+                {
+                    "data": {
+                        "text/markdown": [
+                            "<b>\n",
+                            "Uber's revenue for 2021 was $17,455 million.</b>"
+                        ],
+                        "text/plain": [
+                            "<IPython.core.display.Markdown object>"
+                        ]
+                    },
+                    "metadata": {},
+                    "output_type": "display_data"
+                }
+            ],
+            "source": [
+                "# Provide URI to constructor, or use environment variable\n",
+                "import pymongo\n",
+                "from llama_index.vector_stores.mongodb import MongoDBVectorStore\n",
+                "from llama_index.indices.vector_store.base import VectorStoreIndex\n",
+                "from llama_index.storage.storage_context import StorageContext\n",
+                "from llama_index.readers.file.base import SimpleDirectoryReader\n",
+                "\n",
+                "\n",
+                "# mongo_uri = os.environ[\"MONGO_URI\"]\n",
```

Comment:
nit: this should probably be broken into a few different cells

Reply:
Fixed

Reply:
Looks good!